core/kazoo_im/src/kapps_im_command.erl

```diff
@@ -100,31 +100,12 @@ send_and_wait(API, Endpoint, Timeout) ->
     Payload = props:set_values( [{<<"Endpoints">>, [Endpoint]} | Options], API),
```

Comment:
`Options = kz_json:to_proplist(kz_json:get_value(<<"Endpoint-Options">>, Endpoint, [])),`
Why the to_proplist call on what is presumably a list? Or should the default be `kz_json:new()`?